stablehlo/reference/Element.cpp

```diff
@@ -585,6 +585,22 @@ Element areApproximatelyEqual(const Element &e1, const Element &e2) {
                                      debugString(type).c_str()));
 }
 
+Element atan2(const Element &e1, const Element &e2) {
+  auto type = e1.getType();
+  if (isSupportedFloatType(e1.getType()))
+    return Element(type, std::atan2(e1.getFloatValue().convertToDouble(),
+                                    e2.getFloatValue().convertToDouble()));
+
+  if (isSupportedComplexType(type)) {
```

Comment:
In the spec we mentioned that atan2 for complex computes "complex atan2". Is "complex atan2" and the associated formula ([as mentioned in HLO evaluator](https://github.com/tensorflow/tensorflow/blob/487b79112fa1d8d9afd70a6785a06fa179109cd4/tensorflow/compiler/xla/hlo/evaluator/hlo_evaluator_typed_visitor.h#L438)) a standard textbook thing?